Makefile

```diff
@@ -3,6 +3,7 @@ all:
 
 .PHONY: test
 test:
+	.rocks/bin/luatest -v
```

Comment:
This commit contains changes for README. Change in Makefile should be moved to commit 
"ci: installation, caching and running luatest".

Reply:
Moved

Reply:
The same: no luatest-style tests in a project. You add luatest-style tests later in further commits.